examples/advanced/filters/build.js

```diff
@@ -0,0 +1,32 @@
+const StyleDictionary = require('../../../index');
```

Comment:
Can we remove this file completely since it has nothing really to do with filters?

Reply:
Or if we want to keep this and show adding custom filters and make the file like this example config/build file: https://github.com/amzn/style-dictionary/blob/main/examples/advanced/custom-parser/sd.config.js

Reply:
done the second option -> adjust custom filters